### PR TITLE
CLI-531: As a devops I would like to store vault credentials in SSM only

### PR DESCRIPTION
**Pull request by zivshits**
- Removed store of password in S3 bucket

### CODE REVIEW

File: aws/DRVault-Single-Deployment.json [hunk 1]
```diff
@@ -182,7 +75,7 @@
       "StorePasswordLambda":{
          "Type":"AWS::Lambda::Function",
          "Properties":{
-            "Description":"Encrypts given string and saves chipper to bucket and parameter store",
+            "Description":"Encrypts given string and saves chipper to parameter store",
```
Comment (maishsk) [inline aws/DRVault-Single-Deployment.json]: What is a chipper?

File: aws/Vault-Single-Deployment.json [hunk 2]
```diff
@@ -823,7 +636,7 @@
                                  "'/PrimaryVaultIP',",
                                  "'1.1.1.1',",
                                  "'/DRPassword',",
-                                 "drp,",
+                                 "adminp,",
```
Comment (maishsk) [inline aws/Vault-Single-Deployment.json]: Is this correct ? why did the parameter change?

Comment (zivshits) [reply aws/Vault-Single-Deployment.json]: In vault single deployment there is no need in dr password. There was an object which was created during deployment to store Admin password in DR section and then used here. I have simplified it to use the same password only here without creating a new object.

File: aws/DRVault-Single-Deployment.json (hunk 1 above)
Comment (maishsk) [reply aws/DRVault-Single-Deployment.json]: I understand that this should be cipher ??? 

If so can we please fix?

File: aws/Vault-Single-Deployment.json (hunk 2 above)
Comment (zivshits) [reply aws/Vault-Single-Deployment.json]: @maishsk - fixed StorePasswordLambda desciption

**Pull request merged**